com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeAsset.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace UnityEngine.Experimental.Rendering
 {
+    [PreferBinarySerialization]
```

Comment:
Disclaimer about that: Don't expect too much because by default all unity projects are using the serialization mode "Force text" which bypasses this attribute. Son unless someone changes the setting, there is no effect.

Reply:
At least our template doesn't, I noticed on my project cloned from it that it still serialized as binary, so there is hope :D